src/MagnumPlugins/UfbxImporter/UfbxImporter.conf

```diff
@@ -0,0 +1,2 @@
+provides=FbxImporter
```

Comment:
Here you need

```ini
depends=AnyImageImporter
```

as well, to make the plugin manager correctly load the `AnyImageImporter` before this one. (Maybe *that* was the issue you had with dynamic plugins after all?)

Reply:
Ah thanks! Not sure if this would fix the issue as I had the same problem with `AssimpImporter` even though both worked for a while just fine.

Reply:
What kind of error, or what error message / assertion / etc do you get?

Reply:
I'm debugging within VS2022 and the output window just has consecutive lines of `Loaded dll, Unloaded dll`, `LoadLibraryW()` fails with error 126 (module not found) even though the file is right there. In the past I've had issues with this when having missing .dll dependencies, I tried to use dependency walker to examine the .dll but it crashed.

I'll look into it later but I'm probably just making a mess with CMake as I'm not super familiar with it yet.

Reply:
Huh. No idea, especially if it worked before.

If static plugins work well enough for you, go with that for now. The CI will test the rest.